### DISCUSSION TITLE
Is bulk delete possible?

### DISCUSSION THREAD

**Discussion by raulrpearson** [pocketbase/pocketbase#3356] [Q&A] [answered]
It it possible to delete several records in one go? Right now it seems I can only [do this](https://pocketbase.io/docs/api-records/#delete-record) one at a time.

**Comment by ganigeorgiev** [comment, marked as answer]
No, the web APIs don't support bulk and transactional operations at the moment (it is in the roadmap and will be implemented as part of #48 but it is temporary left aside for now).

**Reply by raulrpearson** [reply]
thanks 🙏
